### ISSUE TITLE
Git GUI: Selecting Target Directory via "Browse" causes "Location <name> already exists." #820

### ISSUE THREAD

**Issue by HaoSun1** [git-for-windows/git#2511]
- [ ] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

git version 2.25.0.windows.1
cpu: x86_64
built from commit: 7c71c859c97853ed057da5cbab12f3c13b5554df
sizeof-long: 4
sizeof-size_t: 8
$ git --version --build-options

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

Microsoft Windows [Version 10.0.18363.592]
$ cmd.exe /c ver

 - What options did you set as part of the installation? Or did you choose the
   defaults?

Nano editor 
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: Nano
Custom Editor Path:
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: ConHost
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Disabled

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

No

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Git GUI

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
Start Git GUI.
Select Clone Existing Repository.
Choose a folder
```
 - What did you expect to occur after running these commands?

a git repository to be created

 - What actually happened instead?

error  target directory already exists

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?
N.A.

Basically same question as https://github.com/git-for-windows/git/issues/820   the question is closed and the workaround doesn't work for me.

**Comment by dscho** [comment]
Please do insert your responses in all the forgotten places.

**Comment by HaoSun1** [comment]
updated in case this wasn't clear it's caused by the git GUI asks where target directory I choose one and then it says   Location ... already exists  Idk what to do I have to choose a target directory and any target directory causes this error.

**Comment by dscho** [comment]
> Basically same question as #820 the question is closed and the workaround doesn't work for me.

#820 was closed because it was determined that this is not a Windows-specific issue. Or did you see anything Windows-specific about it?

**Comment by HaoSun1** [comment]
I guess I don't know if this is windows specific,  can you move this post or should I make a new one?

**Comment by dscho** [comment]
> I guess I don't know if this is windows specific, can you move this post or should I make a new one?

The Git GUI project does not welcome GitHub issues. You will have to send a plain-text-only mail to git@vger.kernel.org, Cc'ing @prati0100

**Comment by PhilipOakley** [comment]
@HaoSun1 to clarify a little.. 
It is the upstream Git project (which contains a copy of git-gui) that does not use GitHub issues. 
There is now a Git Gui maintainer (@prati0100, https://github.com/prati0100/git-gui) who can take GitHub issues, but broader discussion does happen on the Git list.

If you can check if the issue occurs in Linux / WSL then that would clarify the issue ownership. 

Providing a clear detailed MVCE (with copied error messages) will make a world of difference, including aspects such as trailing directory separators, and potential confusion about repeated directory names (perhaps being the same as project name).

Hope that helps.

**Comment by prati0100** [comment]
Hi @HaoSun1,

What you need to do is to set the location as the folder in which you want the repo code to exist. So, for example you want to clone a repo called `foo`, you should set the path to something like `~/src/foo`, and not `~/src/`.

Git (command line) creates that directory automatically for you, but git-gui does not. I agree that it is a usability problem, so I'll see if I can come up with a patch to fix it. In the meantime, just specify the repo name along with the path.
